README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+---
+title: "hw01"
+author: "Frank"
+date: "4/8/2020"
+output: html_document
+---
+
+# My personal introduction
+
+## Brief Introduction
+
+My name is Feiyang **Frank** Yu. I am from MAPSS program, majored in political science and sociology. I am from the southern part of China. I have try to learn *Python* by myself two years ago, and this is the first time I learn something about R. It seems to me that the use of R is like using the functions in Python. It is efficient for data scientists, which is very helpful to me as well.
```

Comment:
Python is super helpful and your experience will serve you well as you learn R going forward! Especially in terms of data science which seems to be your focus.